### PR TITLE
feat: send nfts to L2

### LINKED ISSUE #246: ️🛠 - Send NFTs to L2
### Task description

Bloom should be able to send NFTs to L2, since FF supports that feature. It got removed when we moved to the send flow.

### Requirements

- [ ] Send NFTs from L1 to L2. Use https://evm-toolkit.evm.testnet.shimmer.network to see if they have arrived.

### PR DESCRIPTION
**Pull request by amadeu2**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/src/lib/core/layer-2/utils/estimateGasForLayer1ToLayer2Transaction.ts [hunk 1]
```diff
@@ -5,10 +5,6 @@ import { FALLBACK_GAS_BUDGET, ISC_MAGIC_CONTRACT_ADDRESS } from '../constants'
 import { getIscpTransferSmartContractData } from '../utils'
 
 export async function estimateGasForLayer1ToLayer2Transaction(sendFlowParameters: SendFlowParameters): Promise<number> {
-    if (sendFlowParameters.type === SendFlowType.NftTransfer) {
-        return 0
-    }
-
     const { recipient, layer2Parameters } = sendFlowParameters ?? {}
 
     if (!layer2Parameters) {
```
Comment (MarkNerdi) [inline packages/shared/src/lib/core/layer-2/utils/estimateGasForLayer1ToLayer2Transaction.ts:12]: this function needs some update as well. in line 36 we just encode the asset and amount, but if its an nft, we need to encode that as well

File: packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte [hunk 2]
```diff
@@ -173,6 +159,9 @@
             $sendFlowRouter.previous()
         }
     }
+    onMount(() => {
+        buildNetworkRecipientOptions()
+    })
 </script>
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte:207]: `'SubjectType' is not defined`

File: packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte [hunk 3]
```diff
@@ -119,32 +128,9 @@
...
-        const recipients: Subject[] = ContactManager.listContactAddressesForNetwork(networkId).map((address) => {
-            const contact = ContactManager.getContact(address.contactId)
-            return {
-                type: SubjectType.Contact,
-                address: address.address,
-                contact,
-            }
-        })
-        return [...new Map(recipients.map((recipient) => [recipient?.['contact']?.['id'], recipient])).values()]
-    }
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte]: why remove these?
We still need the calculation of the recipients

Comment (MarkNerdi) [reply packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte]: ah i see that nicole did it on another pr

File: packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte (hunk 2 above)
Comment (amadeu2) [reply packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte:207]: Thanks, fixed

**Pull request merged**